web/static/css/app.css

```diff
@@ -0,0 +1,38 @@
+ul {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    height: 55px;
+}
+ul {
```

Comment:
Once we install SASS. http://sass-lang.com/ You would be able to nest css classes.

```css
ul {
    position: fixed;
    top: 0;
    width: 100%;
    height: 55px;
    list-style-type: none;
    margin: 0;
    padding: 0;
    overflow: hidden;
    background-color: #000033;

    li {
        float: left;
        border-right: 2px solid #bbb;

        a {
            display: block;
            color: white;
            text-align: center;
            padding: 15px 16px;
            text-decoration: none;

            &:hover {
                background-color: #C0C0C0;
            }

            &:last-child {
                float: right;
            }
        }
    }
}

.active {
    background-color: #0000FF;
}
```